bitcoin/src/blockdata/weight.rs

```diff
@@ -215,6 +215,12 @@ impl From<Weight> for u64 {
     fn from(value: Weight) -> Self { value.to_wu() }
 }
 
+impl Add<u64> for Weight {
+    type Output = Weight;
```

Comment:
Hmm semantically this is wrong right? You should convert the u64 to weight and add that.

Reply:
Yes I think I see what you mean.  To convert to Weight from u64 we would be explicit about from kwu I think.